TTAPI/C#/7.17.x/OrderFilter/frmOrderFilter.cs

```diff
@@ -281,6 +279,15 @@ void instrRequest_Completed(object sender, InstrumentLookupSubscriptionEventArgs
             }
         }
 
+        private InstrumentTradeSubscription getNewInstrumentTradeSubscription(Instrument pInstrument)
+        {
+            InstrumentTradeSubscription tITSubsctiption = new InstrumentTradeSubscription(m_TTAPI.Session, Dispatcher.Current, pInstrument);
+            tITSubsctiption.OrderAdded += new EventHandler<OrderAddedEventArgs>(instrumentTradeSubscription_OrderAdded);
+            tITSubsctiption.OrderDeleted += new EventHandler<OrderDeletedEventArgs>(instrumentTradeSubscription_OrderDeleted);
+
+            return tITSubsctiption;
+        }
```

Comment:
@Zack-TT Couple things:

1)  Let's rename to CreateInstrumentTradeSubscription and make it return void.  Use the global member variable instead of a temp local variable, so first you'd need to check if it is null and if not call Dispose on the current.

2)  Keep consistent with the variable formatting.  Variables should be in camel case, and member variables prepend the "m_".